### ISSUE TITLE
[Suggestion]: Make an Ultimate Target setting for when you're in other dimensions

### ISSUE THREAD

**Issue by Mattomo101** [nonamecrackers2/crackers-wither-storm-mod#1713]
### Description of feature request

So what I'm looking to be added is an option to make the Wither Storm randomly roam around or do "Random Strolling" when the player is in another dimension. For example, if I go to the Nether, the Wither Storm no longer has me as a target because I'm not in the same dimension. Logically I think it would start roaming around trying to find more mobs and terrain for it to absorb because it can no longer detect the player. So I think an option for that would be really awesome.

**Comment by TaintedBlock** [comment]
technically it already does this tho. Just around the players last location. Although, random strolling nearest makes this possible

**Comment by Mattomo101** [comment]
> technically it already does this tho. Just around the players last location. Although, random strolling nearest makes this possible

Yes, but I'm looking for it to stroll beyond my last location. I want it to go wherever it wants while I'm gone.
